src/Layouts/stars/starsPage/filtersHeader.tsx

```diff
@@ -0,0 +1,306 @@
+import {
+  Table,
+  TableRow,
+  TableCell,
+  TableBody,
+  Button,
+  Chip,
+  TextField,
+} from '@mui/material';
+import {
+  CheckCircleOutline,
+  Search,
+  PersonOutline,
+  DateRange,
+  Flight,
+  MoreVert,
+  FlashOn,
+  Computer,
+} from '@material-ui/icons';
+import { useState } from 'react';
+import {
+  statuses,
+  assignees,
+  versions,
+  resources,
+  computers,
+} from '../../../assets/star';
+
+interface filterProps {
+  setSearchValue: (param: string) => void;
+  statusFilter: string;
+  assigneeFilter: string;
+  versionFilter: string;
+  resourceFilter: string;
+  computerFilter: string;
+  setFilter: (field: string, value: string) => void;
+}
+
+const FiltersHeader = ({
+  setSearchValue,
+  statusFilter,
+  assigneeFilter,
+  versionFilter,
+  resourceFilter,
+  computerFilter,
+  setFilter,
+}: filterProps) => {
+  const [displayOptions, setDisplayOptions] = useState(false);
+  const [search, setSearch] = useState(false);
+  const [options, setOptions] = useState<string[]>([]);
+  const [lastTab, setLastTab] = useState('');
+  const [displayMore, setDisplayMore] = useState(false);
+
+  const filterEmpty = statusFilter === ''
```

Comment:
get all of the filters as one array of object like in selectedFilters then use
`[].some(sf=>sf.filter==="")`

Reply:
fixed